Documentation/Functions/Data.rst

```diff
@@ -797,6 +797,45 @@ Get content from the :ref:`register <using-setting-register>`:
 
     lib.foo.data = register : color
 
+
+..  _data-type-gettext-request:
+
+request
+-------
+
+..  versionadded:: 12.4
+
+..  confval:: request
+    :name: data-request
+    :Syntax: request : [attribute] | [property]
```

Comment:
Just saw the underlyinf issue. One seems to use | not as a single separator but for nested keys too. So maybe write:

request : [attribute] | [property] ( | [subproperty ] | ... )

or something like that? Or say "use | as a separator like you normally do with the . notation?"

Reply:
as common notation I'd see this:

request : [ attribute | property | subproperty ] 

but I think your notation is clear too, just the round brackets really looks like there is an optional aspect ...

Reply:
Actually, ub-properties are not possible (as property would be an array which is not a scalar, so an empty string is returned - the code is not "recursive").

Reply:
Done.